rules/windows/process_creation/proc_creation_win_reg_defender_tampering.yml

```diff
@@ -22,18 +23,38 @@ detection:
         CommandLine|contains:
             - 'SOFTWARE\Microsoft\Windows Defender\'
             - 'SOFTWARE\Policies\Microsoft\Windows Defender\'
+            - 'SOFTWARE\Policies\Microsoft\Windows Defender Security Center' 
     selection_dword_0:
         CommandLine|contains|all:
             - ' add '
             - ' /d 0'
+            - ' -d 0'
```

Comment:
caution it is a `|all` selection so the nex detection is for ` add /d 0 -d 0`

Reply:
Yes. that's why I said i'll take over this PR and fix some stuff :)